packages/dx-react-grid/docs/reference/search-panel.md

```diff
@@ -0,0 +1,48 @@
+# SearchingPanel Plugin Reference
+
+A plugin that renders a searching panel.
+
+## User Reference
+
+### Dependencies
+
+- [SearchingState](searching-state.md)
+- [IntegratedFiltering](integrated-filtering.md)
+- [Toolbar](toolbar.md)
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+rootComponent | ComponentType&lt;[SearchPanel.RootProps](#searchpanelrootprops)&gt; | | A component that renders the search panel root element.
+messages? | [SearchPanel.LocalizationMessages](#localization-messages) | | An object that specifies localization messages.
+
+## Interfaces
+
+### SearchPanel.RootProps
+
+Describes properties passed to a component that renders the search panel root element.
```

Comment:
At the top of this article, you call this plugin `searching panel`. In this place - `search panel`. I think we should call this like `the Searching Panel`.